docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -0,0 +1,128 @@
+
+[!INCLUDE[interactive-note](~/includes/roslyn-installation.md)]
+
+Familiarize yourself with the concepts used in the .NET Compiler Platform SDK by reading the [overview](compiler-api-model.md) article. It provides an introduction to syntax trees, nodes, tokens, and trivia.
+
+## Syntax Visualizer
+
+The **Syntax Visualizer** enables inspection of the syntax tree for the C# or VB code file in the current active editor window inside the Visual Studio IDE. The visualizer can be launched by clicking on **View** > **Other Windows** > **Syntax Visualizer**.  You can also use the **Quick Launch** toolbar in the upper right corner. Type "syntax" and the command to open the **Syntax Visualizer** should appear.
+
+This command opens the Syntax Visualizer as a floating tool window. If you don't have a code editor window open, the display is blank, as shown in the following figure. 
+
+![The Syntax Visualizer tool window](media/syntax-visualizer.png)
+
+Dock this tool window at a convenient location inside Visual Studio, such as the left side. The Visualizer shows information about the current code file.
+
+Create a new project using the **File** > **New Project** command. You can create either a VB or C# project. When Visual Studio opens the main code file for this project, the visualizer displays the syntax tree for it. You can open any existing C# / VB file in this Visual Studio instance, and the visualizer displays that file's syntax tree. If you have multiple code files open inside Visual Studio, the visualizer displays the syntax tree for the currently active code file, the code file that has keyboard focus.
+
+![Visualizing a C# syntax tree](media/visualize-csharp.png)
+
+![Visualizing a VB syntax tree](media/visualize-visual-basic.png)
+
+As shown in the preceding images, the visualizer tool window displays the syntax tree at the top and a property grid at the bottom. The property grid displays the properties of the item that is currently selected in the tree, including the .NET *Type* and the *Kind* (SyntaxKind) of the item.
+
+Syntax trees comprise three types of items – *nodes*, *tokens*, and *trivia*. You can read more about these types in the [Work with syntax](work-with-syntax.md) article. Items of each type are represented using a different color. Click on the button titled ‘Legend’ for an overview of the colors used.
+
+Each item in the tree also displays its own **span**. The **span** is the indices of that node in the text file.  In the preceding C# example, the selected “UsingKeyword [0..5)” token has a **Span** that is five characters wide, [0..5). The "[..)" notation means that the starting index is part of the span, but the ending index is not.
```

Comment:
it may be helpful to define indices -- e.g., "is the indices (the starting and ending position) of that node..."
This paragraph is also a candidate for a tab so the VB and C# text could appear in separate tabs (for VB, "ImportsKeyword [0,7]".